### ISSUE TITLE
LabelBinarizer has confusing names when there are only two labels

### ISSUE THREAD

**Issue by turian** [scikit-learn-contrib/sklearn-pandas#242]
When there are only two labels, transformed_names_ in LabelBinarizer is confusing. Let's say I have pet is "dog" or "cat". I am visualizing the decision tree, and it has splits like "pet < 0.5".  I can't tell if that means cat or dog.

Could there be a special case for LabelBinarizer that if there are only two labels, the label with value 1.0 is in the name?

**Comment by ragrawal** [comment]
This is beyond the scope of sklearn-pandas. We rely on transformers to provide appropriate column names. One option is you can write your own custom LabelBinarizer that correlectly selects the` self.classes_` variable. We use this variable to get category labels.
